cmd/ui/src/views/GroupManagement/GroupManagement.tsx

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import EntityInfoPanel from '../Explore/EntityInfo/EntityInfoPanel';
+import { DropdownOption, GroupManagementContent } from 'bh-shared-ui';
+import { SelectedNode } from 'src/ducks/entityinfo/types';
+import { useState } from 'react';
+import DataSelector from '../QA/DataSelector';
+import { AssetGroup, AssetGroupMember } from 'js-client-library';
+import { GraphNodeTypes } from 'src/ducks/graph/types';
+import { faGem } from '@fortawesome/free-solid-svg-icons';
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
It would be good to import useAppDispatch instead of useDispatch. We need to tighten up some of our typing to truly benefit from this but importing useAppDispatch now means less change would be needed in the future.